### DISCUSSION TITLE
Any News ?

### DISCUSSION THREAD

**Discussion by nivcoo** [BenWoodworth/FastCraft#202] [Q&A]
Hello i used fastcraft 2 and with the 3 i haven't all features. Do you have planned to finish the "migration" ? Thx

**Comment by BenWoodworth** [comment]
Hey!! What're the features that you're missing?

**Comment by nivcoo** [comment]
Hello, localization, i've to edit the lang from the ZIP (Maybe can you add the feature because if you update the plugin we've to open the zip and replace the language file).

Also we cannot jump line in local file lore, we cannot custom step of multiplicator. And the "Localize Item Names" doesn't work yet

**Comment by nivcoo** [comment]
Also, in your wiki you don't have added all permission, for example : fascraft.use to open gui from crafting table

**Comment by nivcoo** [comment]
And maybe add shift-click to craft all result with the items

**Comment by BenWoodworth** [comment]
Sorry for the late reply. A couple years ago I did have the idea to [make v3.3 a localization update](https://github.com/BenWoodworth/FastCraft/milestone/4), making text more easily customizable and translating item names, I just haven't had much free time between a full time job and life being rough in general.

Besides localization though, FastCraft is pretty much where I want it. No specific plans for some of the other "migrations" you mentioned (but feel free to make a separate issue requesting them!)

Could you make a separate issue for each of these features/issues, and I can take a look at them when I get a chance? :)
